src/main/webapp/intrigue-api/metacards/metacards.js

```diff
@@ -460,6 +460,11 @@ const saveMetacard = async (parent, args, context) => {
     )
   }
 
+  if (attributes.queries) {
```

Comment:
:thinking: I think we should consider moving this logic to the react container instead. Normally I would agree that the graphql server should be handling the business logic, but transporting the entire query when you only need the id seems  like a big difference. Do you mind testing the difference on a simulated slow 3g network?